src/helpers/helpers.math.js

```diff
@@ -140,3 +143,60 @@ export function getAngleFromPoint(centrePoint, anglePoint) {
 export function distanceBetweenPoints(pt1, pt2) {
 	return Math.sqrt(Math.pow(pt2.x - pt1.x, 2) + Math.pow(pt2.y - pt1.y, 2));
 }
+
+/**
+ * @private
+ */
+export function _pointInLine(p1, p2, t) {
+	return {
+		x: p1.x + t * (p2.x - p1.x),
+		y: p1.y + t * (p2.y - p1.y)
+	};
+}
+
+/**
+ * @private
+ */
+export function _steppedInterpolation(p1, p2, t, mode) {
+	return {
+		x: p1.x + t * (p2.x - p1.x),
+		y: mode === 'middle' ? t < 0.5 ? p1.y : p2.y
+		: mode === 'after' ? t < 1 ? p1.y : p2.y
+		: t > 0 ? p2.y : p1.y
+	};
+}
+
+/**
+ * @private
+ */
+export function _bezierInterpolation(p1, p2, t) {
+	const cp1 = {x: p1.controlPointNextX, y: p1.controlPointNextY};
+	const cp2 = {x: p2.controlPointPreviousX, y: p2.controlPointPreviousY};
+	const a = _pointInLine(p1, cp1, t);
+	const b = _pointInLine(cp1, cp2, t);
+	const c = _pointInLine(cp2, p2, t);
+	const d = _pointInLine(a, b, t);
+	const e = _pointInLine(b, c, t);
+	return _pointInLine(d, e, t);
+}
+
+/**
+ * @private
+ */
+export function _angleDiff(a, b) {
+	return (a - b + PITAU) % TAU - PI;
+}
+
+/**
+ * @private
+ */
+export function _normalizeAngle(a) {
+	return (a + TAU) % TAU;
+}
+
+/**
+ * @private
+ */
+export function _angleDiffCW(a, b) {
```

Comment:
It's not clear to me what `CW` means. Can we rename or add a doc?

Reply:
It stood for clockwise, but that is not actually absolutely true so renamed and added a comment.